src/containers/Trigger/Trigger.tsx

```diff
@@ -197,8 +198,12 @@ export const Trigger: React.SFC<TriggerProps> = ({ match }) => {
     setStartTime(moment(startAtValue).format('Thh:mm:ss'));
     setfrequency(triggerFrequency.filter((trigger) => trigger.value === frequencyValue)[0]);
     setDaysDisabled(frequencyValue !== 'weekly');
-    setFlowId(flow.flows.filter((flows: any) => flows.id === flowValue.id)[0]);
-    setGroupId(collections.groups.filter((collection: any) => collection.id === groupValue.id)[0]);
+    const getFlowId = flow.flows.filter((flows: any) => flows.id === flowValue.id);
+    const getcollectionId = collections.groups.filter(
+      (collection: any) => collection.id === groupValue.id
+    );
+    setFlowId(getFlowId.length > 0 ? getFlowId[0] : '');
```

Comment:
Can we make it clearer? There's no need to set an empty value.
`
if (getFlowId.length > 0) {
        setFlowId(getFlowId[0]);
      }
`